rero_ils/modules/acq_orders/templates/rero_ils/vendor_order_mail/ita.tpl.txt

```diff
@@ -9,14 +9,24 @@ Vorremmo ordinare la seguente lista di libri:
 
 ------------------------------------------------------------
 {%- for line in order.order_lines %}
-Titolo : {{ line.document.title }}
-{%- if line.document.identifiers %}
-ISBN : {{ line.document.identifiers | join(', ') }}
-{%- endif %}
+Documento :
+  {{ line.document.title_text }}
+  {%- if line.document.provision_activity %}
+  Pubblicazione, manifattura, distribuzione, produzione : {{ line.document.provision_activity }}
```

Comment:
Official translation of RDA (https://original.rdatoolkit.org/)

```suggestion
  Formulazione di pubblicazione : {{ line.document.provision_activity }}
```

Reply:
OK for me